docs/DeveloperGuide.md

```diff
@@ -739,6 +739,97 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Viewing all persons
+#### Viewing all persons successfully
+**Steps:**
+1. Test case: `list`.
+
+**Expected Result:**
+1. *Person List* shows all persons in FINDvisor.
+2. *Command Result Box* outputs a successful execution message.
+3. *Command Box* is cleared.
+
+**Alternative Test Cases:**
+- Test case: `list 1`.
+- Test case: `list a`.
```

Comment:
```suggestion
- `list 1`.
- `list a`.
```
Would this be sufficient? Given that the heading for the list already states Alternative Test Cases? If this is to be changed, changes should be made for the other alternative test cases.

Reply:
@KohGuanZeh Yep I shall change accordingly